Formula/blast.rb

```diff
@@ -17,6 +17,16 @@ class Blast < Formula
   conflicts_with "proj", :because => "both install a `libproj.a` library"
 
   def install
+    # Fix shebang line
+    inreplace "c++/src/app/blast/update_blastdb.pl", "#!/usr/bin/perl", "#!/usr/bin/env perl"
+    # Use JSON::PP instead of JSON, as the former has been a core perl module
+    # since perl 5.14 and thus is more widely available.
+    inreplace "c++/src/app/blast/update_blastdb.pl", "JSON", "JSON::PP"
+    inreplace "c++/src/app/blast/update_blastdb.pl", "from_json", "decode_json"
+
+    # Fix shebang line, for those who have edirect
+    inreplace "c++/src/app/blast/get_species_taxids.sh", /edirect$/, "edirect:$PATH"
```

Comment:
```suggestion
    inreplace "c++/src/app/blast/get_species_taxids.sh",
      "export PATH=/bin:/usr/bin:/am/ncbiapdata/bin:$HOME/edirect",
      ""
```
Does this work?

Reply:
No, it will break other processes.

Reply:
Why is that, and have you tried it?

Reply:
```ruby

inreplace "c++/src/app/blast/get_species_taxids.sh",
  "export PATH=/bin:/usr/bin:/am/ncbiapdata/bin:$HOME/edirect",
  "export PATH='#{bin}:#{HOMEBREW_PREFIX}/bin:/usr/bin:/bin'"
```
Alternatively, you could try the above.

Reply:
Because there are processes that depend on those paths. Yes, I have tried.
Thank you.

Reply:
`/bin:/usr/bin` should exist in the user's `PATH`, and the neither of the directories `/am/ncbiapdata/bin` nor `$HOME/edirect` should exist, so I don't follow `edirect` should depend on those paths. What command did you run, and what was the exact error message?

Reply:
>  neither of the directories /am/ncbiapdata/bin nor $HOME/edirect should exist

They exist in systems I work with. Is there any harm in leaving these alone?
Thank you, regards.

Reply:
No harm really. How about this:
```suggestion
    inreplace "c++/src/app/blast/get_species_taxids.sh", /edirect$/, "edirect:$PATH"
    "export PATH=/bin:/usr/bin:/am/ncbiapdata/bin:$HOME/edirect",
    "export PATH=#{bin}:#{HOMEBREW_PREFIX}/bin:$PATH:/am/ncbiapdata/bin:$HOME/edirect"
```

Reply:
Great, thanks!